### PR TITLE
Added prepent text to tag block

### PR DESCRIPTION
**Pull request by Quintis1212**
Hello ) This is an enhancement for  "Category and tag blocks: Add option to prepend text" #29909 . I added Rich Text component to tag block , but it is not saving changes when I click publish and go to published page. Please , who can help me with saving changes ?

### CODE REVIEW

**Comment by ntsekouras** [comment]
Hey @Quintis1212 - thanks for working on this! 

Can you please setup linter/prettier (https://developer.wordpress.org/block-editor/contributors/code/getting-started-with-code-contribution/#developer-tools)? There are many whitespace changes introduced that are not needed :) 

>I added Rich Text component to tag block , but it is not saving changes

Also without checking thoroughly your code, if we introduce a new block attribute (`byline`) we need to declare it in the block's `block.json` file.

**Comment by Quintis1212** [comment]
> Hey @Quintis1212 - thanks for working on this!
> 
> Can you please setup linter/prettier (https://developer.wordpress.org/block-editor/contributors/code/getting-started-with-code-contribution/#developer-tools)? There are many whitespace changes introduced that are not needed :)
> 
> > I added Rich Text component to tag block , but it is not saving changes
> 
> Also without checking thoroughly your code, if we introduce a new block attribute (`byline`) we need to declare it in the block's `block.json` file.

Hi @ntsekouras ) I installed prettier as you advised to me and add block to json file

File: packages/block-library/src/post-tags/edit.js [hunk 1]
```diff
@@ -1,103 +1,115 @@
...
+				 <AlignmentToolbar
+					 value={ textAlign }
+					 onChange={ ( nextAlign ) => {
+						 setAttributes( { textAlign: nextAlign } );
+					 } }
+				 />
+			 </BlockControls>
+			 <div { ...blockProps }>
+				 <RichText
+					 className="wp-block-post-author__byline"
```
Comment (ntsekouras) [inline packages/block-library/src/post-tags/edit.js:102]: This needs to change to `post-tags` block (`wp-block-post-author__byline` -> `wp-block-post-tags__byline`).

File: packages/block-library/src/post-tags/edit.js [hunk 2]
```diff
@@ -1,103 +1,115 @@
...
+					 multiline={ false }
+					 aria-label={ __( 'Byline text' ) }
+					 placeholder={ __( 'Write byline…' ) }
+					 value={ byline }
```
Comment (ntsekouras) [inline packages/block-library/src/post-tags/edit.js:106]: `keepPlaceholderOnFocus` property could be useful here..

File: packages/block-library/src/post-tags/index.php [hunk 3]
```diff
@@ -33,7 +34,7 @@ function render_block_core_post_tags( $attributes, $content, $block ) {
 		}
 
 		$output  = trim( $output, ' | ' );
-		$output .= '</div>';
+		$output .= '</div>'.( ! empty( $byline ) ? '<p class="wp-block-post-author__byline">' . $byline . '</p>' : '' );
```
Comment (ntsekouras) [inline packages/block-library/src/post-tags/index.php:37]: `byline` output should be inside the block and before the list output, as is in the editor. Also the class name should be about `post-tags`.

**Review by ntsekouras — COMMENTED** [review]
Thanks for following on this! If you've set up linter/prettier, please push the proper changes to  make it easier to review and the CI happy :) (https://github.com/WordPress/gutenberg/pull/30048/checks?check_run_id=2207905167)

The direction is going is good though and we need some polishing here 👍

File: packages/block-library/src/post-tags/edit.js (hunk 1 above)
Comment (Quintis1212) [reply packages/block-library/src/post-tags/edit.js:102]: > This needs to change to `post-tags` block.

What do you mean under "this" ?

**Comment by carolinan** [comment]
Hi!
I am closing this pull request because https://github.com/WordPress/gutenberg/pull/40559 has been merged instead.

**Pull request closed without merging**